### PR TITLE
Returning localPath with strigns.json in extractStringsFromApk

### PR DESCRIPTION
**Pull request by moizjv**

### CODE REVIEW

File: lib/tools/apk-utils.js [hunk 1]
```diff
@@ -163,6 +163,7 @@ apkUtilsMethods.install = async function (apk, replace = true) {
 apkUtilsMethods.extractStringsFromApk = async function (apk, language, out) {
   log.debug("Extracting strings for language: " + (language || "default"));
```
Comment (imurchie) [inline lib/tools/apk-utils.js]: String interpolation?

File: lib/tools/apk-utils.js [hunk 2]
```diff
@@ -180,7 +181,8 @@ apkUtilsMethods.extractStringsFromApk = async function (apk, language, out) {
 
   try {
     log.debug("Reading strings from converted strings.json");
-    fileData = await fs.readFile(path.join(out, stringsJson), 'utf8');
+    localPath = path.join(out, stringsJson);
+    fileData = await fs.readFile(localPath, 'utf8');
```
Comment (imurchie) [inline lib/tools/apk-utils.js]: It looks like this is a local, home-grown `fs`. It would be good to shift to `appium-support`'s version.

Comment (moizjv) [inline lib/tools/apk-utils.js]: i will create an issue for this right now and we can take it up later :)

**Comment by moizjv** [comment]
https://github.com/appium/appium-adb/issues/98

**Pull request merged**